### PR TITLE
Notebooks

### LINKED ISSUE #2: Add notebooks
#### Description
Add Jupyter notebook files

### PR DESCRIPTION
**Pull request by sadrasabouri**
#### Reference Issues/PRs
#2 
#### What does this implement/fix? Explain your changes.
Notebooks added according to datasets.
Some useful functions added to load data from .csv files and plot them.

### CODE REVIEW

File: Notebooks/README.md [hunk 1]
```diff
@@ -11,7 +11,10 @@
 		<td align="center"><b>Travis</b></td>
 		<td align="center"><b>Appveyor</b></td>
 	</tr>
-
+	<tr>
+		<td align="center"><b>master</b></td>
+		<td align="center"><a href="https://travis-ci.org/ECSIM/dbfc-dataset"><img src="https://travis-ci.org/ECSIM/dbfc-dataset.svg?branch=master"></a></td>
+		<td align="center"><a href="https://ci.appveyor.com/project/sepandhaghighi/dbfc-dataset"><img src="https://ci.appveyor.com/api/projects/status/?svg=true"></a></td>
```
Comment (sadrasabouri) [inline Notebooks/README.md]: @sepandhaghighi please add your Appveyor link of pr here :+1: .

File: Notebooks/functions.py [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+          'seagreen', 'goldenrod', 'turquoise', 'darkorange', 'dimgray', 'lawngreen', 'darkblue', 'lime', 'cadetblue', 'mistyrose', 'mediumorchid', 'mediumseagreen', 'lightyellow', 'mediumspringgreen', 'black', 'darkviolet', 'lightskyblue', 'silver', 'maroon', 'darkkhaki', 'aliceblue', 'gray', 'lightgrey', 'darkslategray', 'magenta', 'palegoldenrod', 'steelblue', 'yellow']
+MARKERS = [".","o","o","s","p","*","+","X","|","v","^","H","<",">","1","2","3","x","D","h"] * 5
+
+def load_data(path):
+    """
+    Load dataset.
+
+    :param path: file path
+    :type path: str
+    :return: data as numpy arr
```
Comment (sepandhaghighi) [inline Notebooks/functions.py]: numpy array ;-)

File: Notebooks/functions.py [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+    y_plot_data = []
+    filtered_data = data
+    title = ""
+    if V is not None:
+        filtered_data = filtered_data[filtered_data[:, 2] == V]
+        title += "V: {}(V)".format(V)
+    if SBH is not None:
+        filtered_data = filtered_data[filtered_data[:, 3] == SBH]
+        title += " SBH: {}(%) ".format(SBH)
+    if CL is not None:
```
Comment (sepandhaghighi) [inline Notebooks/functions.py]: I think the first three if conditions (`is not None`) in this function are redundant and you can mix them with next ifs.

File: Notebooks/functions.py [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+    :type CL: float
+    :return: None
+    """
+    x_plot_data = []
+    y_plot_data = []
+    filtered_data = data
+    title = ""
+    if V is not None:
+        filtered_data = filtered_data[filtered_data[:, 2] == V]
+        title += "V: {}(V)".format(V)
```
Comment (sepandhaghighi) [inline Notebooks/functions.py]: try to provide enough space and comma in mixed titles. Example  :
```
title = "V: {}V , P: {}psig".format(str(V), str(P))
```

**Review by sepandhaghighi — CHANGES_REQUESTED** [review]

File: Notebooks/README.md (hunk 1 above)
Comment (sepandhaghighi) [reply Notebooks/README.md]: Done!

**Review by sepandhaghighi — APPROVED** [review]

**Comment by sepandhaghighi** [comment]
@sadrasabouri Well done ;-)
@sarminh @kasraaskari Technically LGTM, please review notebooks :
1. [Acta Catalyst](https://nbviewer.jupyter.org/github/ECSIM/dbfc-dataset/blob/notebooks/Notebooks/Acta%20Catalyst.ipynb)
2. [Pd-C Catalyst](https://nbviewer.jupyter.org/github/ECSIM/dbfc-dataset/blob/notebooks/Notebooks/Pd-C%20Catalyst.ipynb)
3. [Pt-C Catalyst](https://nbviewer.jupyter.org/github/ECSIM/dbfc-dataset/blob/notebooks/Notebooks/Pt-C%20Catalyst.ipynb)

**Comment by sepandhaghighi** [comment]
@sadrasabouri As a final note, would you please do something to clear `, CL : None` in Pt-C Catalyst plots title?

**Comment by sadrasabouri** [comment]
> @sadrasabouri As a final note, would you please do something to clear `, CL : None` in Pt-C Catalyst plots title?

Thank you for note.
It is solved.

**Review by sarminh — APPROVED** [review]
Fine and approved.

**Review by kasraaskari — APPROVED** [review]
It's done.

**Pull request merged**